cpp/include/cudf/io/datasource.hpp

```diff
@@ -22,8 +22,10 @@
 #include <rmm/cuda_stream_view.hpp>
 
 #include <arrow/buffer.h>
+#pragma nv_diag_suppress 611
```

Comment:
```suggestion

// We disable warning 611 because Arrow currently has issue
// "#611-D: overloaded virtual function "arrow::fs::FileSystem::Equals" is only partially overridden"
#pragma nv_diag_suppress 611
```